package.json

```diff
@@ -172,7 +172,7 @@
     "stylelint-config-prettier": "^9.0.3",
     "stylelint-config-standard": "^37.0.0",
     "stylelint-prettier": "^5.0.3",
-    "tailwindcss": "^3.3.3",
+    "tailwindcss": "^3.4.17",
```

Comment:
dare you to go to v4 👀

Reply:
(in a later PR)

Reply:
im tempted lol but will try later once i managed to clean up the css modules